src/mumble/Settings.cpp

```diff
@@ -401,6 +446,8 @@ std::size_t qHash(const ChannelTarget &target) {
 	return qHash(target.channelID);
 }
 
+const QString Profiles::s_default_profile_name = QLatin1String("default");
```

Comment:
Why not use `std::string` for this?

Reply:
@Krzmbrzl Are you happy with the current state of the code facilitating many conversions between ``std::string`` and ``QString``?

I don't want to refactor to ``std:map`` until we are happy with the string class used